### PR TITLE
Tweak CSS to allow wc_input_price class to work in the settings context

### PR DESCRIPTION
**Pull request by nathanss**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This adds CSS to allow the `wc_input_price` class to work correctly in the settings page. We have a demand in an extension to add a price field to the settings page.

Part of https://github.com/woocommerce/woocommerce-min-max-quantities/issues/380

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to WooCommerce > Settings > General and define `,` as the decimal separator. 
2. Install the following snippet that adds a custom price field in the WooCommerce > Settings > Products tab: 

```
add_filter( 'woocommerce_products_general_settings', 'add_settings', 60 );

function add_settings( $settings ) {

	$settings_to_add = array(
		array(
					'title' => __( 'Test section', 'woocommerce' ),
					'type'  => 'title',
					'desc'  => '',
					'id'    => 'product_test_section',
				),
		array(
			'name'     => __( 'Custom Price', 'woocommerce-min-max-quantities' ),
			'desc'     => __( 'A Custom Price field that does nothing.', 'woocommerce-min-max-quantities' ),
			'id'       => 'woocommerce_custom_price_test_field',
			'type'     => 'text',
			'class'    => 'wc_input_price',
			'desc_tip' => true,
		),
		array(
					'type' => 'sectionend',
					'id'   => 'product_inventory_options',
				),
	);
	$new_settings = array_merge( $settings, $settings_to_add );

	return $new_settings;
}
```
3. Go to WooCommerce > Settings > Products and fill in a price with an invalid decimal separator, like `10.5`. 
4. Notice that the error is not aligned to the field. 
5. Pull and build this branch. 
6. Notice that the error is correctly aligned now. 

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/changelog/tweak-forminp-relative [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: enhancement
+Comment: Tweak CSS to allow wc_input_price class to work in the settings context
```
Comment (jimjasson) [inline plugins/woocommerce/changelog/tweak-forminp-relative]:
```suggestion
Comment: Tweak CSS to allow wc_input_price class to work in the WooCommerce > Settings context
```

File: plugins/woocommerce/client/legacy/css/admin.scss [hunk 2]
```diff
@@ -8798,4 +8803,4 @@ html:has(#status-table-templates){
 // Fix for Safari bug: https://bugs.webkit.org/show_bug.cgi?id=280063.
 .woocommerce-admin-page #postbox-container-2 {
 	clear: left;
-}
\ No newline at end of file
+}
```
Comment (jimjasson) [inline plugins/woocommerce/client/legacy/css/admin.scss:8625]: Was this added automatically? If not, let's remove it.

**Review by jimjasson — COMMENTED** [review]
Hey @nathanss, 

Thank you for submitting this PR! 

Given that there isn't any other price field with the `wc_input_price` class in WooCommerce > Settings, I checked the Regular/Sale Price fields under Product Data > General to confirm that they correctly display notices in the right place, when an invalid price is used. I also checked price fields added by the Min/Max Quantities and Product Add-Ons extensions and confirm that they still work as expected. 

That said, I found the testing steps of this PR to be incomplete, as it is not clear what was fixed by your PR. 

Here's what I would write: 
- Go to WooCommerce > Settings > General and define `,` as the decimal separator. 
- Install the following snippet that adds a custom price field in the WooCommerce > Settings > Products tab: 

```
add_filter( 'woocommerce_products_general_settings', 'add_settings', 60 );

function add_settings( $settings ) {

	$settings_to_add = array(
		array(
			'name'     => __( 'Custom Price', 'woocommerce-min-max-quantities' ),
			'desc'     => __( 'A Custom Price field that does nothing.', 'woocommerce-min-max-quantities' ),
			'id'       => 'woocommerce_custom_price_test_field',
			'type'     => 'text',
			'class'    => 'wc_input_price',
			'desc_tip' => true,
		),
	);
	$new_settings = array_merge( $settings, $settings_to_add );

	return $new_settings;
}
```
- Go to WooCommerce > Settings > Products and fill in a price with an invalid decimal separator, like `10.5`. 
- Notice that the error is not aligned to the field. 
- Pull and build this branch. 
- Notice that the error is correctly aligned now.

File: plugins/woocommerce/client/legacy/css/admin.scss (hunk 2 above)
Comment (nathanss) [reply plugins/woocommerce/client/legacy/css/admin.scss:8625]: It was!

**Comment by nathanss** [comment]
@jimjasson Thanks for providing very detailed testing instructions! I updated the post with them :)

**Review by jimjasson — APPROVED** [review]
Looks good now @nathanss -- thank you for updating the testing instructions.

**Pull request merged**